### ISSUE TITLE
Stops reading ArtNet values in + - 5 seconds

### ISSUE THREAD

**Issue by Light-oneNL** [basstronix/UnityArtNet#1]
After pressing play, ArtNet information comes in, but after like 5 seconds it stops reading values.

Any idea whats happening?

`ArtNet received From: 127.0.0.1:6454
UnityEngine.Debug:Log(Object)
ArtDotNet.ArtNetClient:Communicator_DataReceived(Object, UdpPacket) (at Assets/ArtNet/Scripts/ArtNetClient.cs:77)
ArtDotNet.UdpCommunicator:Server_DoWork(Object, DoWorkEventArgs) (at Assets/ArtNet/Scripts/UdpCommunicator.cs:54)
System.Threading._ThreadPoolWaitCallback:PerformWaitCallback()`

**Comment by basstronix** [comment]
Are you able to confirm it's not an issue with the ArtNet host (i.e. does ArtNetominator show data flow)?

**Comment by Light-oneNL** [comment]
> Are you able to confirm it's not an issue with the ArtNet host (i.e. does ArtNetominator show data flow)?

Yes, artnetominator keeps showing output, so that's not the case

**Comment by DimitriCroes** [comment]
It looks like there is a bug in the UdpClient in the .Net framework that makes it stop sending/receiving after a amount of seconds. https://www.codeproject.com/Articles/1938/Issues-with-UdpClient-Receive

**Comment by basstronix** [comment]
Thanks for bringing this to my attention. I hope to find time to sort it out this weekend.

**Comment by DimitriCroes** [comment]
Cool i have the same issue, so if you need someone to test a possible fix let me know. (because i think you aren't getting this issue)
